Formula/zsh-vi-mode.rb

```diff
@@ -0,0 +1,27 @@
+class ZshViMode < Formula
+  desc "Better and friendly vi(vim) mode plugin for ZSH"
+  homepage "https://github.com/jeffreytse/zsh-vi-mode"
+  url "https://github.com/jeffreytse/zsh-vi-mode/archive/refs/tags/v0.8.2.tar.gz"
+  sha256 "888d68b451fc3221c2ee1648dbfbff38e524048b2c78620a244650fc76946c28"
+  license "MIT"
+
+  bottle :unneeded
+
+  def install
+    pkgshare.install "zsh-vi-mode.zsh"
+  end
+
+  def caveats
+    <<~EOS
+      To activate the zsh vi mode, add the following at the end of your .zshrc:
+        source #{HOMEBREW_PREFIX}/share/zsh-vi-mode/zsh-vi-mode.zsh
+      You will also need to force reload of your .zshrc:
+        source ~/.zshrc
```

Comment:
This isn't great advice, as `.zshrc`s can often be not idempotent. Preferably, this should just suggest restarting the terminal.